src/dockleHelper.ts

```diff
@@ -91,10 +107,12 @@ function getDockleOutput(): any {
 
 function getCisSummary(): any {
     const dockleOutputJson = getDockleOutput();
-    let cisSummary = '';
+    let cisSummary = 'Best practices test summary -';
     const dockleSummary = dockleOutputJson['summary'];
     if (dockleSummary) {
-        cisSummary = `Best practices test summary -\n"fatal": ${dockleSummary["fatal"]}\n"warn": ${dockleSummary["warn"]}\n"info": ${dockleSummary["info"]}\n"pass": ${dockleSummary["pass"]}`;
+        for (let level in dockleSummary) {

```

Comment:
Use levels from the level's array maybe? What if the order is skewed.

Reply:
Javascript retains the key order in JSON. It's not required.
References:
https://stackoverflow.com/a/23202095/4089974
https://medium.com/@ty0h/preserving-json-object-keys-order-in-javascript-python-and-go-language-170eaae0de03